src/frontend/components/UI/QuestsViewer/components/QuestDetailsWrapper/index.tsx

```diff
@@ -212,8 +221,16 @@ export function QuestDetailsWrapper({
   const userId = session.data?.userId
   const isSignedIn = !!userId
 
+  const hasMetStreak =
+    questPlayStreakData?.current_playstreak_in_days !== undefined &&
+    questMeta?.eligibility?.play_streak?.required_playstreak_in_days !==
+      undefined &&
+    questPlayStreakData?.current_playstreak_in_days >=
+      questMeta.eligibility?.play_streak?.required_playstreak_in_days
+
```

Comment:
I've always wondered how to handle optional numeric values because I know `0` is falsy, but I've always like to do boolean checks to catch both `undefined` and `null` without having to do 

```js
if (value !== undefined && value !== null)
```

You can use the chain operator for this, although I can agree it looks funny :D 

```js
const hasMetStreak = (questPlayStreakData?.current_playstreak_in_days ?? 0) >= (questMeta?.eligibility?.play_streak?.required_playstreak_in_days ?? Infinity);
```

Reply:
yeah I can change it to that. it will handle the case where `current_playstreak_in_days` and `required_playstreak_in_days` are null. That case shouldn't happen in practice but better to be safe